### ISSUE TITLE
[Bug]: "OCP\Files\\NotPermittedException" with message "Could not create folder" after Nextcloud 25 update 

### ISSUE THREAD

**Issue by cctgteddy** [nextcloud/server#36421]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

 * I've verified permissions are correct on the installation directory (/data/www-root/asco/nextcloud), apache is the owner of the directory and all contents
 * I've put SELinux in permissive mode (though I did confirm the file contexts in https://docs.nextcloud.com/server/latest/admin_manual/installation/selinux_configuration.html)
 * Host is running CentOS  7.9 and PHP 8.1.14 (from Remi's repository)

My page loads aren't as snappy as they should be an I'm suspicious this might be related.  I'm configured to use APCu for local and Redis for distributed.  

Here's the error from nextcloud.log

```json
{"reqId":"Y9Uznjwewm2P28k6WPnIogAAAAs","level":3,"time":"2023-01-28T14:39:26+00:00","remoteAddr":"216.212.202.217","user":"admin","app":"index","method":"GET","url":"/nextcloud/index.php/apps/theming/favicon/settings?v=bf8dad03","message":"Could not create folder","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0","version":"25.0.3.2","exception":{"Exception":"OCP\\Files\\NotPermittedException","Message":"Could not create folder","Code":0,"Trace":[{"file":"/data/www-root/asco/nextcloud/lib/private/Files/SimpleFS/SimpleFolder.php","line":106,"function":"newFolder","class":"OC\\Files\\Node\\Folder","type":"->","args":["18"]},{"file":"/data/www-root/asco/nextcloud/apps/theming/lib/ImageManager.php","line":165,"function":"newFolder","class":"OC\\Files\\SimpleFS\\SimpleFolder","type":"->","args":["18"]},{"file":"/data/www-root/asco/nextcloud/apps/theming/lib/ImageManager.php","line":180,"function":"getCacheFolder","class":"OCA\\Theming\\ImageManager","type":"->","args":[]},{"file":"/data/www-root/asco/nextcloud/apps/theming/lib/Controller/IconController.php","line":124,"function":"getCachedImage","class":"OCA\\Theming\\ImageManager","type":"->","args":["favIcon-settings"]},{"file":"/data/www-root/asco/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":225,"function":"getFavicon","class":"OCA\\Theming\\Controller\\IconController","type":"->","args":["settings"]},{"file":"/data/www-root/asco/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":133,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Theming\\Controller\\IconController"},"getFavicon"]},{"file":"/data/www-root/asco/nextcloud/lib/private/AppFramework/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Theming\\Controller\\IconController"},"getFavicon"]},{"file":"/data/www-root/asco/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Theming\\Controller\\IconController","getFavicon",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},["settings","theming.Icon.getFavicon"]]},{"file":"/data/www-root/asco/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/theming/favicon/settings"]},{"file":"/data/www-root/asco/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/data/www-root/asco/nextcloud/lib/private/Files/Node/Folder.php","Line":166,"CustomMessage":"--"}}
```

### Steps to reproduce

1. Happens on every page load

### Expected behavior

Do not expect this error.

### Installation method

Community Manual installation with Archive

### Operating system

RHEL/CentOS

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "asco.ctg.queensu.ca"
        ],
        "defaultapp": "files",
        "default_phone_region": "CA",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.3.2",
        "overwrite.cli.url": "https:\/\/asco.ctg.queensu.ca\/nextcloud",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "skeletondirectory": "\/data\/nextcloud_data\/skeleton",
        "maintenance": false,
        "loglevel": 2,
        "filelocking.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        }
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.17.0
  - admin_audit: 1.15.0
  - bruteforcesettings: 2.5.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contactsinteraction: 1.6.0
  - dav: 1.24.0
  - federatedfilesharing: 1.15.0
  - files: 1.20.1
  - files_pdfviewer: 2.6.0
  - files_retention: 1.14.1
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - viewer: 1.9.0
  - workflowengine: 2.7.0
Disabled:
  - circles: 23.0.1
  - dashboard: 7.3.0
  - encryption
  - federation: 1.15.0
  - files_external
  - firstrunwizard: 2.14.0
  - nextcloud_announcements: 1.12.0
  - photos: 2.0.1
  - sharebymail: 1.6.0
  - support: 1.0.0
  - suspicious_login
  - twofactor_totp
  - updatenotification: 1.15.0
  - user_ldap: 1.6.0
  - user_status: 1.3.1
  - weather_status: 1.3.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
My logs are too big, I will attach
```

**Comment by cctgteddy** [comment]
[nextcloud3.log](https://github.com/nextcloud/server/files/10527607/nextcloud3.log)

**Comment by szaimen** [comment]
Should hopefully get fixed with 25.0.4

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.
